errors/error.go

```diff
@@ -70,41 +71,41 @@ const separator = ": "
 //
 // The supported types are:
 //
-// 	errors.Kind
-//		The kind of error (eg.: HCLSyntax, TerramateSchema, etc).
-//	hcl.Range
-//		The file range where the error originated.
-//	errors.StackMeta
-//		The stack that originated the error.
-//	hcl.Diagnostics
-//		The underlying hcl error that triggered this one.
-//		Only the first hcl.Diagnostic will be used.
-//		If hcl.Range is not set, the diagnostic subject range is pulled.
-//		If the string Description is not set, the diagnostic detail field is
-//		pulled.
-//	hcl.Diagnostic
-//		Same behavior as hcl.Diagnostics but for a single diagnostic.
-//	string
-//		The error description. It supports formatting using the Go's fmt verbs
-//		as long as the arguments are not one of the defined types.
+//		errors.Kind
```

Comment:
I think this fmt fix is related to Go 1.19 additions on Go doc, now supporting links/lists/etc.